src/GafferScene/SceneAlgo.cpp

```diff
@@ -614,6 +628,31 @@ void addCopyAttributesPredecessors( const CopyAttributes *copyAttributes, const
 	}
 }
 
+void addCopyOptionsPredecessors( const CopyOptions *copyOptions, const SceneAlgo::History::Predecessors &source, SceneAlgo::OptionHistory *destination )
+{
+	const ScenePlug *sourceScene = copyOptions->inPlug();
+	if( StringAlgo::matchMultiple( destination->optionName, copyOptions->optionsPlug()->getValue() ) )
+	{
+		ConstCompoundObjectPtr sourceOptions;
+		if( copyOptions->sourcePlug()->exists() )
```

Comment:
`exists()` is a query to check that the location defined by `scene:path` actually exists, so isn't relevant here. So I think we  want to ditch the `if` and just have `ConstCompoundObjectPtr sourceOptions = copyOptions->sourcePlug()->globalsPlug()->getValue()`